### PR TITLE
Fixes #407: Update open-source overview page

### LINKED ISSUE #407: Update the open-source page
Our [open-source overview](https://www.datarequests.org/open-source/) has become quite outdated, we should update it! 

* We no longer use `yace` but [backend](https://github.com/datenanfragen/) which atm handles comments, suggestions, donations, and hacktoberfest registrations
* There are other (new) repos, such as media, letter-generator, and locate-contacts-addon, should we include these as well?

### PR DESCRIPTION
**Pull request by michizhou**
This PR solves issue #407. Updated information about new repositories and backend in the open-source overview page.

### CODE REVIEW

File: content/en/open-source.md [hunk 1]
```diff
@@ -72,16 +98,16 @@ Here is an overview of important repositories that we have created—directly or
...
 <article class="list-article icon-list-article">
-    <div class="col25 article-featured-image"><a href="https://github.com/baltpeter/yace"><img class="image" src="/card-icons/speech-bubble.svg" alt="yace"></a></div>
+    <div class="col25 article-featured-image"><a href="https://github.com/datenanfragen/backend"><img class="image" src="/card-icons/speech-bubble.svg" alt="backend"></a></div>
     <div class="padded col75">
-        <a href="https://github.com/baltpeter/yace"><h1><code>yace</code></h1></a>
+        <a href="https://github.com/datenanfragen/backend"><h1><code>backend</code></h1></a>
         <span class="license">MIT License</span>
         <p class="description">
-            A simple, privacy-focussed and easy-to-deploy engine to build custom comment solutions, hosted on AWS.
+            A development server that defines endpoints for dynamic features of the website such as comments, suggestions, and donations.
```
Comment (baltpeter) [inline content/en/open-source.md:106]: The term "development server" may be a little confusing as this is also the backend we use in production.

File: content/en/open-source.md [hunk 2]
```diff
@@ -45,6 +45,32 @@ Here is an overview of important repositories that we have created—directly or
...
+    <a class="button button-primary read-more-button" href="https://github.com/datenanfragen/media">View repository&nbsp;<span class="icon icon-arrow-right"></span></a>
+</article>
+
+<article class="list-article icon-list-article">
+    <div class="col25 article-featured-image"><a href="https://github.com/datenanfragen/letter-generator"><img class="image" src="/card-icons/letter.svg" alt="letter-generator"></a></div>
+    <div class="padded col75">
+        <a href="https://github.com/datenanfragen/letter-generator"><h1><code>letter-generator</code></h1></a>
+        <span class="license">MIT License</span>
+        <p class="description">
+            A tool that allows you to generate letters (plain text or PDF formats) from pre-defined templates. This repository includes these templates and the scripts needed to render letters from them.
```
Comment (baltpeter) [inline content/en/open-source.md:67]: I think "templates" could easily be confused with the sample letters in `data` (I admit that this is our fault—we named them both the same :D). Maybe "layout templates" is clearer?

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks, looks good! Just two small change requests below.

**Comment by michizhou** [comment]
@baltpeter I modified the descriptions for the `backend` and `letter-generator` repositories as per your suggestions. Please let me know if there is any other chances that should be made. Thank you.

**Review by baltpeter — APPROVED** [review]
Thanks again!

**Pull request merged**